src/views/About/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+
+import Typography from 'material-ui/Typography';
+import './about.css';
+
+export default class About extends Component {
+  render() {
+    return (
+      <div className="container page-title about-container about">
```

Comment:
Nice job converting all the `class` attributes to `className`.

Since we aren't using any styles in this file anymore, you should be able to remove the `className` attributes on all elements!